tools/cldr-apps/src/test/java/org/unicode/cldr/unittest/web/TestSTFactory.java

```diff
@@ -545,6 +560,36 @@ public void handlePathValue(String path, String value) {
                                                     + xpath);
                                 }
                                 break;
+                            case "apivote":
+                            case "apiunvote":
```

Comment:
minor nitpick, but there's a comment above, "TODO: shorten this function, over 300 lines; use subroutines" -- and this makes about 10% longer

Reply:
refactored to subroutines

Reply:
for me, regardless of what it is that makes a function long, if it won't fit on the screen all at once without scrolling, it's too long; 300+ lines is waaaay too long

granted, breaking into subroutines may not always be worth the difficulty/effort; this is important code, though...